### ISSUE TITLE
Helm chart is not downloadable

### ISSUE THREAD

**Issue by rafernandez** [kubernetes-sigs/blob-csi-driver#569]
Hello !

**What happened**:

Helm chart is not downloadable

**What you expected to happen**:

Helm chart is downloadable

**How to reproduce it**:
```
helm repo add blob-csi-driver https://raw.githubusercontent.com/kubernetes-sigs/blob-csi-driver/master/charts
helm repo update

helm search repo blob-csi-driver
NAME                               	CHART VERSION	APP VERSION	DESCRIPTION                                       
blob-csi-driver/blob-csi-driver    	v1.6.0       	v1.6.0     	Azure Blob Storage CSI driver                     
blob-csi-driver/blobfuse-csi-driver	v0.6.0       	v0.6.0     	BlobFuse Container Storage Interface (CSI) Stor...

helm pull --version v1.6.0 blob-csi-driver/blob-csi-driver
Error: failed to fetch https://raw.githubusercontent.com/kubernetes-sigs/blob-csi-driver/master/charts/latest/blob-csi-driver-v1.6.0.tgz : 404 Not Found

helm pull blob-csi-driver/blob-csi-driver
Error: failed to fetch https://raw.githubusercontent.com/kubernetes-sigs/blob-csi-driver/master/charts/latest/blob-csi-driver-v1.6.0.tgz : 404 Not Found
```

**Anything else we need to know?**:

The version seems to have changed from 1.0.6 to **v**1.0.6

**Environment**:
- CSI Driver version: 1.6.0

Thanks !

**Comment by andyzhangx** [comment]
pls run `helm repo update`, it should work now, fixed by #571

**Comment by rafernandez** [comment]
Thanks, its works
